tools/sideloading/README.md

```diff
@@ -0,0 +1,229 @@
+## Overview
+
+**Sideloading** tool is a sample solution to add side-loading capability to your Panorama application. With this solution, you can modify script files & model files, transfer them to the application code container on the device, and run the updated scripts without re-deploying the entire application. It gives you quicker development iteration experience.
+
+This solution comes with 1) sideloading agent module you can import in your Panorama application, 2) sideloading commandline interface you can use on your development host.
+
+## Features
+
+* Automatically identify updated files and transfer them. (**"sync" command**)
+* Run the updated scripts and models as a process. (**"run-app" command**)
+* Kill the process. (**"kill-app" command**)
+* Easily switch the application from side-loading enabled development mode to production mode, with single line of code change.
+
+
+## Security notice
+
+* > We do not recommend doing this on a production system because this creates a security vulnerability and might disrupt the normal operation of your device.
```

Comment:
... because this is a debugging feature that has not been AWS Security Validated for use in production systems.

Reply:
Updating :

> We do not recommend doing this on a production system because this is a development feature that has not been AWS Security validated for use in production systems.